src/main/views/serverDropdownView.ts

```diff
@@ -83,6 +84,7 @@ export class ServerDropdownView {
             // @ts-ignore
             transparent: true,
         }});
+        performanceMonitor.registerView('ServerDropdownView', this.view.webContents);
```

Comment:
nit: for this and all the other registered views, maybe put the names in global constant values.
```suggestion
        performanceMonitor.registerView(SERVER_DROPDOWN_VIEW, this.view.webContents);
```

Reply:
This would be worth doing I'd say if we were using these constants in multiple places, but right now they're just labels that get passed along with the data and will only matter once it hits the dashboard, so I'm okay to leave this one as is.